internal/backuprestore/backup.go

```diff
@@ -41,27 +41,27 @@ import (
 )
 
 // ReconcileBackup reconciles the backup CRs
-func (h *BRHandler) ReconcileBackup(ctx context.Context, oadpContent []lcav1alpha1.ConfigMapRef,
-) (
-	result ctrl.Result, status BackupStatus, err error,
-) {
+func (h *BRHandler) ReconcileBackup(ctx context.Context, oadpContent []lcav1alpha1.ConfigMapRef) (ctrl.Result, BackupStatus, error) {
+	var (
+		status = BackupStatus{}
+		result = ctrl.Result{}
+	)
 	if len(oadpContent) == 0 {
-		status.Status = BackupFailedValidation
-		status.Message = "No oadp configmap is provided."
-		result.RequeueAfter = 1 * time.Minute
-		return
+		status.Status = BackupCompleted
+		status.Message = "No oadp configmap is provided. Skipping backup"
+		return result, status, nil
```

Comment:
Add an info log here to indicate no oadp configmap is provided.  For this case, the upgrade will continue and we don't need to update the IBU status.

Reply:
no more logging as discussed